src/main/java/com/pusher/client/connection/websocket/WebSocketClientWrapper.java

```diff
@@ -51,27 +52,29 @@ public WebSocketClientWrapper(final URI uri, final WebSocketListener proxy) thro
                 throw new SSLException(e);
             }
         }
-
-        this.proxy = proxy;
+        this.webSocketListener = webSocketListener;
+        if (proxy != null){
```

Comment:
Space between `)` and `{` please.

Maybe we should consider defaulting `PusherOptions.proxy` to `java.net.Proxy.NOPROXY` rather than `null` and removing this conditional.